src/components/Utility/InputCopy.vue

```diff
@@ -1,6 +1,20 @@
 <template>
-  <b-input-group class="copy-input-container d-flex align-items-center" :size="size">
-    <b-form-input ref="copy-input-field" type="text" class="copy-input" readonly v-model="value"></b-form-input>
+  <b-input-group
+    class="copy-input-container align-items-center"
+    :class="autoWidth ? 'd-inline-flex auto-width mx-1' : 'd-flex'"
+    :style="{
+      width: autoWidth ? `${(10 + value.toString().length) * 8}px` : 'auto',
```

Comment:
Kudos for implementing this brilliant hack! Tweaked the "magical" numbers a little bit. It's still not perfect, but does the job, haha. Made it a prop so we can just add `auto-width` to `input-copy`.